### PR TITLE
FE105

### PR DESCRIPTION
**Pull request by mikesoylu**
HTML CSS DOM

### CODE REVIEW

**Comment by mikesoylu** [comment]
@Pisano/developers anyone wanna review thiz?

File: FE105.md [hunk 1]
```diff
@@ -0,0 +1,771 @@
...
+    Welcome to Pisano!
+  </body>
+</html>
+```
+
+## Elements
+
+Elements provide structure to HTML documents and without them Web pages would
+only consist of text. There are many element types in HTML so we will only go
+thru the most common essentials.
```
Comment (norydev) [inline FE105.md]: through

File: FE105.md [hunk 2]
```diff
@@ -0,0 +1,771 @@
...
+</style>
+
+This is some text. <span class="red">This is some red text</span>
+```
+
+### IDs
+
+Although the `id` attribute is not used as often in modern web apps, its primary
+purpose is to link HTML elements to JavaScript. This link is exposed to JavaScript
+via the Document Object Model or DOM.
```
Comment (norydev) [inline FE105.md]: maybe also as anchors in urls?

**Comment by norydev** [comment]
@mikesoylu  I would add somewhere:
- How to include your js and css files in the html head
- a small word about coffeescript and less/scss?

**Comment by mikesoylu** [comment]
@NoryDev orraydi thx

**Comment by mikesoylu** [comment]
TODO add `box-sizing: border-box;`

**Comment by omurturan** [comment]
We can use this image somewhere :) 

Image of Yaktocat

**Pull request merged**